crates/admin/src/metadata_api/mod.rs

```diff
@@ -125,13 +126,27 @@ async fn put_key(
         .to_str()?
         .parse()?;
 
+    let version = version.into();
     let versioned = VersionedValue {
-        version: version.into(),
-        value,
+        version,
+        value: value.clone(),
     };
 
-    store.put(key, versioned, precondition.into()).await?;
-    Ok(())
+    let result = store
+        .put(key.clone(), versioned, precondition.clone().into())
+        .await;
+    trace!(
+        "Put key: {:?} @{} condition: {:?} value: {} -> result: {}",
+        key,
+        version,
+        precondition,
+        String::from_utf8_lossy(&value),
```

Comment:
I am wondering if the value should be printed in the trace logs. it's probably not an issue during testing but in general this can be big and also binary. Maybe drop the value from the logs and just print the len of it. This can save us a `.clone()` as well of the value.

Reply:
I was specifically after dumping the value in the logs to save me from also needing to dig through packet captures to correlate some Jepsen runs with the server history. It hasn't actually been useful in practice so I'll just take the body logging out completely.